integrations/quarkus/deployment/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 https://maven.apache.org/xsd/maven-4.0.0.xsd"
+         xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+    <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+        <groupId>one.microstream</groupId>
+        <artifactId>microstream-extension-parent</artifactId>
```

Comment:
This should be 
microstream-integration-quarkus-parent

We had discussed Extension / Integration before because all our artifacts are more extensions than integration. Still, we have decided to keep integration to prevent breaking changes for all future integration/extensions.

Reply:
I used the naming of Quarkus (where you have extensions). I updated the name.